addons/binding/org.openhab.binding.somfytahoma/ESH-INF/thing/gate.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="somfytahoma" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
```

Comment:
Switch to OH namespace (see https://github.com/openhab/openhab2-addons/pull/5026#issuecomment-476709809).

Reply:
done